BE/OwnershipAndControl/Executives.rdf

```diff
@@ -257,7 +262,36 @@
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:label>board member</rdfs:label>
-		<skos:definition>party that is a member of the board of directors of an organization</skos:definition>
+		<skos:definition>party that is a director of an organization</skos:definition>
```

Comment:
Perhaps, but 'board of directors' implies that the members are known as directors.  I've removed it, but the term 'director' is definitely used in banks to refer to members of the board.  It may be used for other purposes in other contexts as well, which is why 'director' isn't the name of the concept, but the term is certainly used interchangeably with board member.

Reply:
As per the Wikipedia article, the term "director" is broader than "board member". The latter implies the former but not vice versa. Hence my point to not use (unqualified) "director" in the definition of "board member". "Member of board of directors" is clear albeit circular in this context.